website/docs/docs/collaborate/explore-projects.md

```diff
@@ -51,75 +47,107 @@ By default, dbt Explorer shows the project's [applied state](/docs/dbt-cloud-api
-- `access:` &mdash; Find models based on their [access](/reference/node-selection/methods#the-access-method) property.
+</expandable>
 
-</VersionBlock>
+### Example of lenses
 
-When searching with selector methods, you can also use [graph operators](/reference/node-selection/graph-operators). For example, `+orders` returns all the upstream nodes of `orders` and `orders+1` returns just the nodes immediately downstream (the children) of `orders`.
+Example of applying the **Materialization Type** _lens_ with the lineage graph significantly zoomed out: 
 
-You can use multiple selector methods in your search query with [set operators](/reference/node-selection/set-operators). A space implies a union set operator and a comma for an intersection. For example:
-- `resource_type:metric,tag:nightly` &mdash; Returns metrics with the tag `nightly`
-- `+snowplow_sessions +fct_orders` &mdash; Returns resources that are parent nodes of either `snowplow_sessions` or `fct_orders`
+<Lightbox src="/img/docs/collaborate/dbt-explorer/example-materialization-type-lense.png" width="100%" title="Example of the Materialization type lens" />
 
-<LoomVideo id='712c8b74a6e242699ad2d32e2b0693a5' />
 
-## Browse with the sidebar
+## Keyword search {#search-resources}
+
+You can locate resources in your project by performing keyword search in the search bar. All resource names, column names, resource descriptions, warehouse relations, and code that match your search criteria will display as a list on the main (center) section of the page. When searching for an exact column name, the results show all relational nodes containing that column in their schemas. If there's a match, a notice in the search result indicates the resource contains the specified column. Also, you can apply filters to further refine your search results.
+
+<expandable alt_header="Search features">
+
+- **Partial keyword search** &mdash; This is also referred to as [fuzzy search](https://en.wikipedia.org/wiki/Approximate_string_matching).
+- **Exclude keywords** &mdash; Prepend a minus sign (-) to the keyword you want to exclude from search results. For example, `-user` will exclude all matches of that keyword from search results.
+- **Boolean operators** &mdash; Use Boolean operators to enhance your keyword search. For example, the search results for `users OR github` will include matches for either keyword.
+- **Phrase search** &mdash; Surround a string of keywords with double quotation marks to search for that specific phrase. For example, the search results for `"stg users"` will include matches that contain that specific phrase. To learn more, refer to [Phrase search](https://en.wikipedia.org/wiki/Phrase_search) on Wikipedia.
+- **SQL keyword search** &mdash; Use SQL keywords in your search. For example, the search results `int github users joined` will include matches that contain that specific string of keywords (similar to phrase searching).
```

Comment:
Do we need to define supported SQL keywords? For example, looking at [this resource](https://www.w3schools.com/sql/sql_ref_keywords.asp) and `GitHub` is not on there.

Reply:
good callout! 

this is still beta and i'm anticipating more changes coming soon. let's monitor this and when Explorer GAs we can lean toward that level of quality